tests/ProjectTemplatesTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Keboola\ManageApiTest;
+
+use Keboola\ManageApi\ClientException;
+
+class ProjectTemplatesTest extends ClientTestCase
+{
+    const TEST_PROJECT_TEMPLATE_STRING_ID = 'production';
+
+    const TEST_HIDDEN_PROJECT_TEMPLATE_STRING_ID = 'poc15DaysGuideMode';
+
+    private $organization;
+    /**
+     * Create empty organization without admins, remove admins from test maintainer and delete all their join requests
+     */
+    public function setUp()
+    {
+        parent::setUp();
+        $this->client->addUserToMaintainer($this->testMaintainerId, ['email' => 'spam+spam@keboola.com']);
+        foreach ($this->client->listMaintainerMembers($this->testMaintainerId) as $member) {
+            if ($member['id'] === $this->normalUser['id']) {
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+            if ($member['id'] === $this->superAdmin['id']) {
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+        }
+        $organizations = $this->client->listMaintainerOrganizations($this->testMaintainerId);
+        foreach ($organizations as $organization) {
+            $this->client->deleteOrganization($organization['id']);
+        }
+        $this->organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => 'spam+spam@keboola.com']);
+        $this->client->removeUserFromOrganization($this->organization['id'], $this->superAdmin['id']);
+    }
+
+    public function testSuperAdminCanViewAndListProjectTemplates()
+    {
+        $templates = $this->client->getProjectTemplates();
+
+        $this->assertGreaterThan(0, count($templates));
+
+        $filteredTemplates = array_filter($templates, function ($item) {
+            if ($item['id'] !== self::TEST_PROJECT_TEMPLATE_STRING_ID) {
+                return false;
+            }
+            return true;
+        });
+        $this->assertGreaterThan(0, count($filteredTemplates));
+
+        $templateDetail = $this->client->getProjectTemplate(self::TEST_PROJECT_TEMPLATE_STRING_ID);
+
+        $this->assertEquals($templateDetail, current($filteredTemplates));
+
+    }
+
+    public function testOrganizationAdminCanViewAndListProjectTemplates()
+    {
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => $this->normalUser['email']]);
+
+        $templates = $this->normalUserClient->getProjectTemplates();
+
+        $filteredTemplates = array_filter($templates, function ($item) {
```

Comment:
To muzes napsat taky takhle a bude to prehlednejsi.

```php
        $filteredTemplates = array_filter($templates, function ($item) {
            return $item['id'] === self::TEST_PROJECT_TEMPLATE_STRING_ID;
        });
        $this->assertGreaterThan(1, $filteredTemplates);
```

Reply:
upravil jsem to